README.md

```diff
@@ -1,7 +1,38 @@
-# Material-UI X
+<p align="center">
+  <a href="https://material-ui.com/" rel="noopener" target="_blank"><img width="150" src="https://material-ui.com/static/logo.svg" alt="Material-UI logo"></a></p>
+</p>
+
+<h1 align="center">Material-UI X</h1>
 
 The Material-UI X mono-repo gathers Material-UI advanced components.
 
 ## Components
 
 - [DataGrid](https://material-ui.com/components/data-grid/)
+
+## Questions
+
+For _how-to_ questions and other non-issues,
+please use [StackOverflow](https://stackoverflow.com/questions/tagged/material-ui) instead of GitHub issues.
+There is a StackOverflow tag called "material-ui" that you can use to tag your questions.
+
```

Comment:
Should we link to the paid version with reference to the included support?

Reply:
The "Pro" plan, the only one we sell yet, has no dedicated support attached to it.

Reply:
But maybe we can clarify it here, yes +1

Reply:
> The "Pro" plan, has no dedicated support attached to it

According to the wireframe it comes with a year of support.

Reply:
I was referring to:

<img width="699" alt="Capture d’écran 2020-11-05 à 21 34 09" src="https://user-images.githubusercontent.com/3165635/98293485-ae103380-1fae-11eb-94e7-bb8407799834.png">

in which I believe we provide support as GitHub issues :)

Reply:
So was I, but let's park it for another time.

Reply:
Considering Pro have priority on GitHub, I 👍  the original comment.